### PR TITLE
DietPi on VMware installation description added

### PR DESCRIPTION
**Pull request by StephanStS**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -239,6 +238,60 @@ Select the following tabs for the installation description of your target.
     Press the start button (green arrow) to 'boot up' your system based on the DietPi image.
     If you do not have a wired LAN connection you have to change the network settings matching to your environment (files `\boot\dietpi.txt` and `\boot\dietpi-wifi.txt`).
 
+=== "Install in VMware Player"
+
+    <font size="+2">Introduction</font>
+
+    Virtual machine images are great for those occasions where you want to set up a DietPi system very quickly and test things. Also it may be used as a Debian based Linux system with a small footprint for development purposes, e.g. with the X11 window system. The small footprint makes it optimally usable on PCs without a huge built in RAM. Also several VMs may be run for different applications.
+
+    One big advantage of such a VM is that id needs only a couple of minutes coming to a running DietPi system.
```
Comment (MichaIng) [inline docs/install.md:247]: `id` => `it`

reminds me of my:

`bid` => `bit` 😄

File: docs/install.md [hunk 2]
```diff
@@ -239,6 +238,60 @@ Select the following tabs for the installation description of your target.
...
+
+    ![DietPi VMware 7zip archive content](assets/images/dietpi-VMware-7zip-file.png)
+
+    <font size="+2">2. Add the files in VMware</font>
+
+    As next, the VMware virtual machine is setup by just opening the `.vmx` file (via ***Open a Virtual Machine***):
+
+    ![VMware file open screenshot](assets/images/dietpi-VMware-import1.png)
+
+    In the following dialog the user has to navigate to the directory where the `.vmx` and `.vdmk` file were stored. Then choose DietPi_VMware-x86_64-Buster(`.vmx`) as the file which shall be opened.
```
Comment (MichaIng) [inline docs/install.md:285]: `.vdmk` => `.vmdk`

**Comment by MichaIng** [comment]
Double click on the `.vmx` btw works as well. LGTM aside of the two little typos I found.

File: docs/install.md (hunk 2 above)
Comment (StephanStS) [reply docs/install.md:285]: Ok, changed.

File: docs/install.md (hunk 1 above)
Comment (StephanStS) [reply docs/install.md:247]: Ok. Changed (also at VirtualBox).

**Comment by StephanStS** [comment]
> 
> 
> Double click on the `.vmx` btw works as well. LGTM aside of the two little typos I found.

Yes, I also use klicking .vmx. 
Also editing the vmx file after copying a virtual machine is a quick option to duplicate VMs.

**Pull request merged**